examples-routes.js

```diff
@@ -0,0 +1,10 @@
+module.exports = [
+  { path: '/', title: 'useModal example' },
+  { path: '/prevent-scroll', title: 'useModal with preventScroll example' },
+  { path: '/close-button', title: 'useModal with closeButton example' },
+  {
+    path: '/close-button/render-option',
+    title: 'useModal with renderCloseButton option example',
+  },
+  { path: '/modal-config', title: 'useModal with ModalConfig example' },
+];
```

Comment:
The route information used in `webpack.config.js` and `examples/src/index.tsx` must be the same, so we cut them out.